src/service.ts

```diff
@@ -1,22 +1,31 @@
 import irisSetup from '@repositive/iris';
+import { inject } from '@repositive/iris';
+
 import * as config from 'config';
-import annotate from './annotate';
+import { ingestion, search } from './controller';
+import { Pool } from 'pg';
 
 const pack = require('../package.json');
 
 export default async function init({
   _config = config,
   _irisSetup = irisSetup,
-  _pack = pack
+  _pack = pack,
+  _Pool = Pool
 }: {
-  _config ? : typeof config,
-  _irisSetup ? : typeof irisSetup,
-  _pack ? : { version: string }
-}): Promise < void > {
-  const irisOpts = _config.get < any > ('iris');
+    _config?: typeof config,
+    _irisSetup?: typeof irisSetup,
+    _pack?: { version: string },
+    _Pool?: typeof Pool
+  }): Promise<void> {
 
+  const irisOpts = _config.get<any>('iris');
   const iris = await _irisSetup(irisOpts);
 
+  const pgOpts = _config.get<any>('db');
+  const postgres = new _Pool(pgOpts);
+  postgres.on('error', (err: Error) => console.error(err.stack));
```

Comment:
If this happens... Blow it up.

This basically means "I don't have a database", logging it may hide the problem (at least until we have a proper logging system in place)

So instead of logging do a `process.exit(1)`, this will kill the process and kubernetes will shout and try to spawn a new one. This has the benefit of letting k8s taking care of healing the connection and making it visible for operations.